infiniband/assets/configuration/spec.yaml

```diff
@@ -1,4 +1,4 @@
-name: Infiniband
+name: "Infiniband"
```

Comment:
```suggestion
name: "InfiniBand"
```

Reply:
I can't change this sadly. The validation fails when I try doing so. I didn't want to chase down this change.

Reply:
Weird! Fair enough, disregard whichever of my edits the robots dislike 🙂